src/main/java/org/teamresistance/frc/command/GrabGear.java

```diff
@@ -0,0 +1,24 @@
+package org.teamresistance.frc.command;
+
+import org.strongback.command.Command;
+import org.strongback.components.Solenoid;
+import org.teamresistance.frc.subsystem.GearGrabber.Grabber;
```

Comment:
Do you think we should rename the package from `GearGrabber` to `grabber` for simplicity and consistency?